chapter/simpleTypes.tex

```diff
@@ -15,6 +18,11 @@ \section{Simply Typed Lambda Calculus}
     \tag{values}
 \end{alignat*}
 
+In contrast to evaluation relation, typing relation does include lambda terms
+and to type the body of a lambda term, we must remember current variable binding.
+As lambda terms can be nested, we must store somewhere a set of assumptions about free variables occurring in a term.
+This place is called context $\Gamma$. \\
```

Comment:
I prefer to use "environment" instead of "context", because "context" is very overloaded: we have evaluation contexts, and when we will talk about contextual equivalence we will have "contexts" as terms with a hole, not necessarily at the evaluation position.